src/Symfony/Component/VarExporter/LazyGhostTrait.php

```diff
@@ -193,17 +193,37 @@ public function &__get($name): mixed
 
         get_in_scope:
 
-        if (null === $scope) {
-            if (null === $readonlyScope) {
-                return $this->$name;
+        try {
+            if (null === $scope) {
+                if (null === $readonlyScope) {
+                    return $this->$name;
+                }
+                $value = $this->$name;
+
+                return $value;
```

Comment:
Isn't this block (198-203) just a very complicated way to say `return $this->name;`?

Reply:
It's not :)
The function returns by reference. The first "if" *does* return a reference. The 2nd return does *not* return a reference.